docs/riff_core_deployer.md

```diff
@@ -12,7 +12,7 @@ Deployers can be created for a build or an image. Build based deployers
 continuously watch for the latest image and will deploy new images. If the
 underlying build is deleted, the deployer will continue to run, but will no
 longer self update. Image based deployers must be manually updated to trigger
-roll out of an updated image.
+the rollout of an updated image.
```

Comment:
@glyn thought the two word variant was correct. See https://github.com/projectriff/cli/pull/27#discussion_r301645362

Reply:
I agree with "the rollout" here. "roll out" is a verb and "rollout" is a noun.

Reply:
What you link to is a verb, which would indeed be `[to] roll out`.
In my understanding of the sentence above, this is a noun hence `[the] rollout`.